src/screens/punch-list/punch-list-screen.js

```diff
@@ -1,12 +1,60 @@
 import React from "react";
-import { Text, View } from "react-native";
+import { View, FlatList } from "react-native";
+import PunchCard from "./compose/punch-card/punch-card";
 import Template from "../../components/template/template";
 
+const DATA = [
+  {
+    id: 1,
+    projectName: "Ifood",
+    morningFrom: "09:00",
+    morningTo: "12:00",
+    afternoonFrom: "13:00",
+    afternoonTo: "18:00",
+    totalHours: "08:00",
+    date: "01/10/2021",
+  },
+  {
+    id: 2,
+    projectName: "Ifood",
+    morningFrom: "09:00",
+    morningTo: "12:00",
+    afternoonFrom: "13:00",
+    afternoonTo: "18:00",
+    totalHours: "08:00",
+    date: "01/10/2021",
+  },
+  {
+    id: 3,
+    projectName: "Ifood",
+    morningFrom: "09:00",
+    morningTo: "12:00",
+    afternoonFrom: "13:00",
+    afternoonTo: "18:00",
+    totalHours: "08:00",
+    date: "01/10/2021",
+  },
+  {
+    id: 4,
+    projectName: "Ifood",
+    morningFrom: "09:00",
+    morningTo: "12:00",
+    afternoonFrom: "13:00",
+    afternoonTo: "18:00",
+    totalHours: "08:00",
+    date: "01/10/2021",
+  },
+];
+
 const PunchListScreen = () => {
   return (
     <Template>
-      <View style={{ flex: 1, justifyContent: "center", alignItems: "center" }}>
-        <Text style={{ margin: "auto" }}>Listagem de punches</Text>
+      <View style={{ flex: 1, justifyContent: "center" }}>
+        <FlatList
+          data={DATA}
+          renderItem={PunchCard}
+          keyExtractor={(item) => item.id}
```

Comment:
```suggestion
          keyExtractor={(item) => item.id}
          contentContainerStyle={{ paddingBottom: 70 }}
```
Podemos adicionar um padding bottom no container da lista para que as cards nao colidam com o botão flutuante